l1-contracts/contracts/common/Ownable2StepRandomStorage.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.8.0) (access/Ownable2Step.sol)
+
+pragma solidity ^0.8.0;
+
+import "./OwnableRandomStorage.sol";
+
+/**
+ * @dev Contract module which provides access control mechanism, where
+ * there is an account (an owner) that can be granted exclusive access to
+ * specific functions.
+ *
+ * By default, the owner account will be the one that deploys the contract. This
+ * can later be changed with {transferOwnership} and {acceptOwnership}.
+ *
+ * This module is used through inheritance. It will make available all functions
+ * from parent (Ownable).
+ */
+abstract contract Ownable2StepRandomStorage is OwnableRandomStorage {
+    /// @dev Address of pending owner variable.
+    /// @dev Flag is placed at random memory location to not interfere with Storage contract.
+    // keccak256("pending_owner") - 1;
+    uint256 private constant PENDING_OWNER_ADDRESS = 0x44b2657a0f8a90ed8e62f4c4cceca06eacaa9b4b25751ae1ebca9280a70abd67;
+
+    function _getPendingOwner() internal view returns (address _pendingOwner) {
+        assembly {
+            _pendingOwner := sload(PENDING_OWNER_ADDRESS)
+        }
+        return _pendingOwner;
```

Comment:
You can remove this line, since there is named return `returns (address _pendingOwner)`